# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1320397260 14400
#      Fri Nov 04 05:01:00 2011 -0400
# Node ID 72875370e2d1381301f7a1b56ba0e5bdd54c7f66
# Parent  d3f0d75faf2c46b382aca5a0705e4a008b9ffd25
allow waitbar test to succeed even if figure 1 exists

* waitbar.m: Use struct() instead of 1 as second argument in error
test.

diff --git a/scripts/plot/waitbar.m b/scripts/plot/waitbar.m
--- a/scripts/plot/waitbar.m
+++ b/scripts/plot/waitbar.m
@@ -136,11 +136,11 @@ endfunction
 %! for i = 0:0.01:1
 %!   waitbar (i);
 %! endfor
 %! close (h);
 
 %% Test input validation
 %!error <FRAC must be between 0 and 1> waitbar (-0.5)
 %!error <FRAC must be between 0 and 1> waitbar (1.5)
-%!error <MSG must be a character string> waitbar (0.5, 1)
+%!error <MSG must be a character string> waitbar (0.5, struct ())
 %!error <invalid number of property-value pairs> waitbar (0.5, "msg", "Name")
 
